packages/web-runtime/src/components/Topbar/UserMenu.vue

```diff
@@ -93,7 +117,86 @@ export default defineComponent({
     }
   },
   setup() {
+    const checkboxValue = ref(false);
+    const notificationsModalOpen = ref(false)
+    const store = useStore()
+    const accessToken = useAccessToken({ store })
+
+    const updateNotificationPreference = (option) => {
+      const headers = new Headers()
+      headers.append('Authorization', `Bearer ${unref(accessToken)}`)
+      headers.append('X-Requested-With', 'XMLHttpRequest')
+
+      fetch('/ocs/v1.php/cloud/user', {
+        method: 'PATCH',
+        headers,
+        body: JSON.stringify({ disableNotifications: option })
+      })
+        .then((response) => {
+          if (response.ok) {
+            store.commit('SET_NOTIFICATION', option)
+            closeNotificationsModal()
+          } else {
+            throw new Error('Notification setting could not be applied')
+          }
+        })
+        .catch((err) => {
+          store.dispatch('showMessage', {
+            title: 'An error occurred',
+            desc: err || 'Notification setting could not be applied',
```

Comment:
Maybe you can pass this string to `$getText` for it to translate. Not sure if it will work inside of `setup` tho.